app.js

```diff
@@ -21,4 +23,45 @@ myVariable = String(myVariable);
 console.log(myVariable);
 
 myVariable = Boolean(myVariable);
-console.log(myVariable);
\ No newline at end of file
+console.log(myVariable);*/
+
+/*second homework*/ 
+//1
+let text = "";
+for(let i = 1; i<=100;i++){
+    
+    if (i%3 === 0 && i%5 === 0 ){
+        text+= " LOLKEK"
+    }
+       else if (i%3 === 0){
+        text+= " KEK"
+    }        
+        else if (i%5 === 0){
+            text+= " LOL"
+
+        }
+        else{ 
+            text+= ` ${i}`}
+}
+console.log(text);
+let text1 = "";
+let text2 = "";
+//2
+const value = 10;
+let i = 0;
+if (Number.isNaN(value) === false){
```

Comment:
Оскільки isNaN поверне нам булеве значення, ми можемо зразу цей результат використати в умові
```suggestion
if (!Number.isNaN(value)){
```

Reply:
підкоректую у 3 домашці